xml/ha_virtualization.xml

```diff
@@ -0,0 +1,484 @@
+        <para>
+           In the file <filename>/etc/libvirt/qemu-lockd.conf</filename>, find and enable the
+           <literal>file_lockspace_dir</literal> setting and change the value to point to a
+           directory on the &ocfs; volume:
+        </para>
+<screen>file_lockspace_dir = "/mnt/shared/lockd"</screen>
+      </step>
+      <step>
+        <para>
+          Enable and start the <literal>libvirtd</literal> service:
+        </para>
+<screen>&prompt.root;<command>systemctl enable --now libvirtd</command></screen>
+        <para>
+          This also starts the <literal>virtlockd</literal> service.
+        </para>
+      </step>
+    </procedure>
+  </sect1>
+
+  <sect1 xml:id="sec-ha-virtualization-adding-virtual-machines">
+    <title>Adding virtual machines as cluster resources</title>
+    <para>
+      Repeat this procedure for each virtual machine you want to create.
+    </para>
+    <procedure xml:id="pro-ha-virtualization-adding-virtual-machines">
+      <title>Adding virtual machines as cluster resources</title>
+      <step>
+        <para>
+          Install a virtual machine on one of the cluster nodes, with the following restrictions:
+        </para>
+        <itemizedlist>
+          <listitem>
+            <para>
+              The installation source and storage must be on shared devices.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              Do not configure the VM to start on host boot.
+            </para>
+          </listitem>
+        </itemizedlist>
+        <para>
+          For more information, see
+          <link xlink:href="https://documentation.suse.com/sles/html/SLES-all/book-virtualization.html">
+          <citetitle>&virtual;</citetitle> for &sles;</link>.
+        </para>
+      </step>
+      <step>
+        <para>
+          If the virtual machine is running, shut it down. The cluster will start the VM
+          after you add it as a cluster resource.
+        </para>
+      </step>
+      <step>
+        <para>
+          Dump the virtual machine's XML configuration to a file on the &ocfs; volume:
+        </para>
+<screen>&prompt.root;<command>virsh dumpxml vm1 &gt; /mnt/shared/vm1.xml</command></screen>
+        <important role="compact">
+          <para>
+            Make sure the XML file does not contain any references to unshared local paths.
+          </para>
+        </important>
+      </step>
+      <step>
+        <para>
+          Run <command>crm configure</command> to start the <command>crm</command> interactive shell.
+        </para>
+      </step>
+      <step>
+        <para>
+          Create a primitive resource to manage the virtual machine:
+        </para>
+<screen>&prompt.crm.conf;<command>primitive vm1 VirtualDomain \
+  params config="/mnt/shared/vm1.xml" remoteuri="qemu+ssh://%n/system" \
+  meta allow-migrate=true \
+  op monitor timeout=30s interval=10s \
+  utilization cpu=2 hv_memory=1024</command></screen>
```

Comment:
I like the idea to leave out the utilization settings and point to the load impact section, thanks. 

For `autoset_utilization_hv_memory`, I think I'll leave that out as it's only a recommendation rather than a requirement, and it's explained in the same load impact section we'll be linking to.